ambassador/tests/test_ambassador.py

```diff
@@ -750,6 +750,20 @@ def check(self):
                 actual = lowercased_headers.get(k.lower())
                 assert actual == [v], "expected %s: %s but got %s" % (k, v, lowercased_headers)
 
+class RemoveResponseHeaders(OptionTest):
+
+    parent: Test
+    header_to_be_dropped = "x-envoy-upstream-service-time"
+
+    def config(self):
+        yield "remove_response_headers: %s" % header_to_be_dropped
+
+    def check(self):
+        for r in self.parent.results:
+
```

Comment:
Instead of the `for` loop you can probably just do

```
assert r.headers.get(header_to_be_dropped, None) == None, "%s header was meant to be dropped but wasn't" % header_to_be_dropped
```